package.json

```diff
@@ -4,11 +4,12 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "build": "ng build ngx-script-loader --prod",
+    "build": "ng build ngx-script-loader --prod && npm run copy-readme",
```

Comment:
Maybe that should be a part of publish process, not build process 🤔

Reply:
I am fine either way. :)